dotcom-rendering/README.md

```diff
@@ -56,6 +56,19 @@ You can view the JSON representation of an article, as per the model sent to the
 
 http://localhost:3030/ArticleJson?url=https://www.theguardian.com/sport/2019/jul/28/tour-de-france-key-moments-egan-bernal-yellow-jersey
 
+### Environment Variables
+
+| Name                          | Description                                                                                                                                    |
+| ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DEVELOPMENT_HOSTNAME`        | Defaults to `localhost`. If running behind a reverse proxy (Github Codespaces / Ngrok) this needs to be set to the hostname used to access DCR |
```

Comment:
I wanted to make it clear that it was a variable only used in `development`, you wouldn't want to set this if you were running in prod. Happy to change it thought!

Reply:
Subjectively I find that distinction more confusing. "If you set `HOSTNAME` it will use `HOSTNAME` or `localhost`" over "If you set `DEVELOPMENT_HOSTNAME` we will use this in places that are only development". 

I'm not sure if we have a way of setting env vars globally, but it's really useful, and if we expect `HOSTNAME` to always be set, we could just use that in the code not having to remember the rule above.

p.s. bikeshedding.

Reply:
The shed in question is here: #4122